Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -11159,26 +11301,26 @@ def main():
         elif command == 'url':
             if USE_URL_FILTERING:  # default is false
                 panorama_get_url_category_command(url_cmd='url', url=args.get('url'),
-                                                  additional_suspicious=additional_suspicious,
-                                                  additional_malicious=additional_malicious)
+                                                    additional_suspicious=additional_suspicious,
+                                                    additional_malicious=additional_malicious)
             # do not error out
 
         elif command == 'panorama-get-url-category' or command == 'pan-os-get-url-category':
             panorama_get_url_category_command(url_cmd='url', url=args.get('url'),
-                                              additional_suspicious=additional_suspicious,
-                                              additional_malicious=additional_malicious,
-                                              target=args.get('target'),
-                                              )
+                                                additional_suspicious=additional_suspicious,
+                                                additional_malicious=additional_malicious,
+                                                target=args.get('target'),
+                                                )
 
         elif command == 'panorama-get-url-category-from-cloud' or command == 'pan-os-get-url-category-from-cloud':
             panorama_get_url_category_command(url_cmd='url-info-cloud', url=args.get('url'),
-                                              additional_suspicious=additional_suspicious,
-                                              additional_malicious=additional_malicious)
+                                                additional_suspicious=additional_suspicious,
+                                                additional_malicious=additional_malicious)
 
         elif command == 'panorama-get-url-category-from-host' or command == 'pan-os-get-url-category-from-host':
             panorama_get_url_category_command(url_cmd='url-info-host', url=args.get('url'),
-                                              additional_suspicious=additional_suspicious,
-                                              additional_malicious=additional_malicious)
+                                                additional_suspicious=additional_suspicious,
+                                                additional_malicious=additional_malicious)
```

Comment:
please revert

Reply:
I can't see what was changed? According to compare with master branch, they are the same.

Reply:
ill check it for you

Reply:
fixed